README.md

```diff
@@ -116,6 +116,39 @@ logger::log_threshold("DEBUG", namespace = "shiny.telemetry")
 
 _note_: This command can be run before the Shiny call or by adding it to the `.Rprofile`.
 
+## Contributing
+_*Have you read the [Contributing Guidelines](https://github.com/Appsilon/.github/blob/main/CONTRIBUTING.md)?*_
+
+### pre-commit
```

Comment:
This seems like a very long section to maintain across repositories. Could we add this to the contribution guide and only keep a small reference there?

Reply:
I have a feeling that even with this section the hooks will have not-so-great adoption and with moving this information further away we are decreasing our chances. The wording would be awkward too and make it contributors responsibility to check given repository whether it uses pre-commit hooks.

What do you think about a dedicated `CONTRIBUTING.md` file? This would shadow the organization-wide file, but can still reference it at the very top. However, this would boost visibility thanks to GitHub mentioning these in its interface and it would allow to declutter the readme.

Reply:
I like the idea of adding `CONTRIBUTING.md` (with reference in the main README and possibly a page added to the documentation). But also I'm ok with it as the next step and keeping the extended README for now.